src/phoenix.js

```diff
@@ -64,6 +66,7 @@ Vue.use(VueMeta, {
 
 Vue.component('drag', Drag)
 Vue.component('drop', Drop)
+Vue.component('avatar', Avatar)
```

Comment:
Vue style guide suggests to use component names with at least two words / fragments, to avoid clashes with pure html tags (which are always single worded), also to support readability (`it's not single-word, so it can't be html` as someone who is new to phoenix). I suppose that would be a good idea here as well. 🤷‍♂